### PR TITLE
Feature/transfer ownership

### PR DESCRIPTION
**Pull request by alexcos20**
Started from Matthias work from #370 

Requires https://github.com/oceanprotocol/aquarius/pull/230

### CODE REVIEW

File: src/aquarius/Aquarius.ts [hunk 1]
```diff
@@ -215,6 +215,51 @@ export class Aquarius {
         return this.retrieveDDO(undefined, metadataServiceEndpoint)
     }
 
+    /**
+     * Transfer ownership of a DDO
+     * @param  {DID | string} did DID of the asset to update.
+     * @param  {String} newOwner New owner of the DDO
+     * @param  {String} updated Updated field of the DDO
+     * @param  {String} signature Signature using updated field to verify that the consumer has rights
```
Comment (ssallam) [inline src/aquarius/Aquarius.ts]: This should be the signature of the original owner.

File: src/ocean/OceanAssets.ts [hunk 2]
```diff
@@ -412,14 +412,37 @@ export class OceanAssets extends Instantiable {
      * Transfer ownership of an asset.
      * @param  {string} did Asset DID.
      * @param  {string} newOwner Ethereum address of the new owner of the DID.
+     * @param  {Account} account Ethereum account to sign and prove the ownership.
```
Comment (ssallam) [inline src/ocean/OceanAssets.ts]: account of the original/old owner.

File: src/ocean/utils/SignatureUtils.ts [hunk 3]
```diff
@@ -40,4 +41,35 @@ export class SignatureUtils {
     public async verifyText(text: string, signature: string): Promise<string> {
         return this.web3.eth.personal.ecRecover(text, signature)
     }
+
+    public async getHash(message: string): Promise<string> {
+        let hex = ''
+        for (let i = 0; i < message.length; i++) {
+            hex += '' + message.charCodeAt(i).toString(16)
```
Comment (ssallam) [inline src/ocean/utils/SignatureUtils.ts]: There are simpler methods to create the hash. Please take a look at web3.sha3 for example

File: src/aquarius/Aquarius.ts (hunk 1 above)
Comment (alexcos20) [reply src/aquarius/Aquarius.ts]: using updated field as a message for signing, so after the call ends, the signature cannot be reused again

File: src/ocean/OceanAssets.ts (hunk 2 above)
Comment (alexcos20) [reply src/ocean/OceanAssets.ts]: solved

File: src/ocean/utils/SignatureUtils.ts (hunk 3 above)
Comment (alexcos20) [reply src/ocean/utils/SignatureUtils.ts]: tried that, but it fails in python while doing ec.recover.
this was the only thing that worked for squid-js + pyton

**Pull request closed without merging**